code/modules/modular_computers/hardware/card_slot.dm

```diff
@@ -1,21 +1,19 @@
 /obj/item/computer_hardware/card_slot
 	name = "RFID card slot"
 	desc = "Slot that allows this computer to write data on RFID cards. Necessary for some programs to run properly."
-	power_usage = 10 //W
+	power_usage = 10 // W
 	critical = FALSE
 	icon_state = "cardreader"
 	hardware_size = 1
 	origin_tech = list(TECH_DATA = 2)
 
 	var/obj/item/card/id/stored_card
 
-	// Storing items (We PDAs now boys)
-	var/list/allowed_items = list(/obj/item/pen,
-								   /obj/item/lipstick,
-								   /obj/item/device/flashlight/pen,
-								   /obj/item/clothing/mask/smokable/cigarette
-								   )
-	var/obj/item/stored_item //Used for pen, crayon, and lipstick insertion/removal
+	// Storing Items
+	var/list/allowed_items = list(
+		/obj/item/pen
+	)
```

Comment:
Readd the other objects. Unsure why you removed those.

Reply:
addressed.